main.tf

```diff
@@ -1,3 +1,15 @@
+#
+# empty provider to pass `terraform validate`
+# will be overrided by parent in real execution
+# https://github.com/hashicorp/terraform/issues/21416
+#
+provider "aws" {
+  alias = "cloudvision"
+}
+
```

Comment:
this part is no longer required (think git merge trolled us :()  will remove it after merge, no worries :)